.envrc

```
@@ -0,0 +1,15 @@
+strict_env
+
+if ! find_up .venv/pyvenv.cfg; then 
```

```suggestion
if [ ! -d ".venv" ]; then 
```
I know that newer versions of Python create `pyvenv.cfg`, but this didn't always happen to be the case. It's better to just check if the directory for the venv exists.